vars/cloudFoundryDeleteService.groovy

```diff
@@ -0,0 +1,48 @@
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.BashUtils
+import com.sap.piper.PiperGoUtils
+import com.sap.piper.JenkinsUtils
+import com.sap.piper.Utils
+import com.sap.piper.ConfigurationHelper
+
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field def STEP_NAME = getClass().getName()
+@Field String METADATA_FILE = 'metadata/cloudFoundryDeleteService.yaml'
+
+void call(Map parameters = [:]) {
+    handlePipelineStepErrors(stepName: STEP_NAME, stepParameters: parameters, failOnError: true) {
+
+        def script = checkScript(this, parameters) ?: this
+
+        Map config
+        def utils = parameters.juStabUtils ?: new Utils()
+
+        script.commonPipelineEnvironment.writeToDisk(script)
+
+        writeFile(file: METADATA_FILE, text: libraryResource(METADATA_FILE))
+
+        withEnv([
+            "PIPER_parametersJSON=${groovy.json.JsonOutput.toJson(parameters)}",
+        ]) {
+            // get context configuration
+            config = readJSON (text: sh(returnStdout: true, script: "./piper getConfig --contextConfig --stepMetadata '${METADATA_FILE}'"))
+            // execute step
+            dockerExecute(
+                script: script,
+                dockerImage: config.dockerImage,
+                dockerWorkspace: config.dockerWorkspace
+            ) {
+                withCredentials([usernamePassword(
+                    credentialsId: config.cfCredentialsId,
+                    passwordVariable: 'PIPER_password',
+                    usernameVariable: 'PIPER_username'
+                )]) {
+                    sh "./piper cloudFoundryDeleteService"
+                }
+            }
+        }
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
New Line at the end of the file has been added. Didn't think that would throw a warning in CodeClimate though.